scripts/flash/dispute-contribute.js

```diff
@@ -0,0 +1,106 @@
+var BigNumber = require("bignumber.js");
+var displayTime = require("./display-time");
+var setTimestamp = require("./set-timestamp");
+
+var day = 108000; // day
+
+function disputeContributeInternal(augur, marketID, outcome, amount, disputerAuth, invalid, auth, callback) {
+  repFaucet(augur, disputerAuth, function (err) {
+    if (err) {
+      console.log(chalk.red("Error"), chalk.red(err));
+      callback(err);
+    }
+    if (err) return console.error(err);
+    invalid = !invalid ? false : true;
+    augur.markets.getMarketsInfo({ marketIDs: [marketID] }, function (err, marketsInfo) {
+      var market = marketsInfo[0];
+      var marketPayload = { tx: { to: marketID } };
+      augur.api.Market.getFeeWindow(marketPayload, function (err, feeWindowId) {
+        var feeWindowPayload = { tx: { to: feeWindowId } };
+        augur.api.FeeWindow.getStartTime(feeWindowPayload, function (err, feeWindowStartTime) {
+          getTime(augur, auth, function (timeResult) {
+            var setTime = parseInt(feeWindowStartTime, 10) + day;
+            displayTime("Current timestamp", timeResult.timestamp);
+            displayTime("Fee Window end time", feeWindowStartTime);
+            displayTime("Set Time to", setTime);
+            setTimestamp(augur, setTime, timeResult.timeAddress, auth, function () {
+              var numTicks = market.numTicks;
+              var payoutNumerators = Array(market.numOutcomes).fill(0);
+              payoutNumerators[outcome] = numTicks;
+              var bnAmount = new BigNumber(amount, 10).toFixed();
+              console.log(chalk.yellow("sending amount REP"), chalk.yellow(bnAmount));
+              augur.api.FeeWindow.isActive(feeWindowPayload, function (err, result) {
+                console.log(chalk.green.dim("Few Window is active"), chalk.green(result));
+                if (result) {
+                  augur.api.Market.contribute({
+                    meta: disputerAuth,
+                    tx: { to: marketID  },
+                    _payoutNumerators: payoutNumerators,
+                    _invalid: invalid,
+                    _amount: bnAmount,
+                    onSent: function (result) {
+                      console.log(chalk.yellow.dim("Sent Dispute:"), chalk.yellow(JSON.stringify(result)));
+                      console.log(chalk.yellow.dim("Waiting for reply ...."));
+                    },
+                    onSuccess: function (result) {
+                      console.log(chalk.green.dim("Success Dispute:"), chalk.green(JSON.stringify(result)));
+                      callback(null);
+                    },
+                    onFailed: function (result) {
+                      console.log(chalk.red.dim("Failed Dispute:"), chalk.red(JSON.stringify(result)));
+                      callback(result);
+                    },
+                  });
+                } else {
+                  console.log(chalk.red("Fee Window isn't active"));
```

Comment:
Should `callback` be called here?

Reply:
This was changed, don't know why the change isn't reflected here. should be

                 } else {
                  console.log(chalk.red("Fee Window isn't active"));
                  callback("Fee Window isn't active");